pkl-core/src/main/java/org/pkl/core/ast/expression/member/InvokeMethodVirtualNode.java

```diff
@@ -36,6 +36,7 @@
 import org.pkl.core.runtime.VmFunction;
 
 /** A virtual method call. */
+@SuppressWarnings("DuplicatedCode")
```

Comment:
What does this overlap with?

Reply:
The various specializations here. Adding just one more line to each method triggered the duplicate code detection warning.